### ISSUE TITLE
Inconsistent highlighting in org mode

### ISSUE THREAD

**Issue by ssfrr** [syl20bnr/spacemacs#9950]
#### Description :octocat:

I'm seeing strange highlighting in my .org files. The same string (e.g. `DONE` or `TODO` is getting highlighted in different ways, based on rules I can't figure out. It also seems to be stateful in some way, because I can delete a line and then re-add it, and the highlighting will change.

You can see an example below of what it looks like:

![different highlighting](https://user-images.githubusercontent.com/734386/33537365-3275bc82-d889-11e7-8c60-e4efdad98f0c.png)

#### Reproduction guide :beetle:
 (I'm so far not able to create a reliable MWE - perhaps with the info below you can provide some pointers towards one?)

*Observed behaviour:* :eyes: :broken_heart:

`DONE` text shows up in 2 different styles, one bright and one pale.

I ran SPC U SPC SPC what-cursor-position for both the brightly-colored and the paler version, and got the folloring output. The main difference seems to be that in the bright colored version the face is listed as `org-done`, whereas in the paler one it is `(:inherit hl-todo :foreground "#afd8af")`.

##### Bright "DONE"

```
             position: 5088 of 8193 (62%), column: 5
            character: N (displayed as N) (codepoint 78, #o116, #x4e)
    preferred charset: ascii (ASCII (ISO646 IRV))
code point in charset: 0x4E
               script: latin
               syntax: w 	which means: word
             category: .:Base, L:Left-to-right (strong), U:Uppercase, a:ASCII, l:Latin, r:Roman
             to input: type "C-x 8 RET 4e" or "C-x 8 RET LATIN CAPITAL LETTER N"
          buffer code: #x4E
            file code: #x4E (encoded by coding system utf-8-unix)
              display: by this font (glyph code)
    xft:-ADBO-Source Code Pro-bold-normal-normal-*-18-*-*-*-m-0-iso10646-1 (#x0F)

Character code properties: customize what to show
  name: LATIN CAPITAL LETTER N
  general-category: Lu (Letter, Uppercase)
  decomposition: (78) ('N')

There are text properties here:
  face                 org-done
  fontified            t
  org-category         "todo"
```

##### Pale "DONE"

```
             position: 5183 of 8193 (63%), column: 5
            character: N (displayed as N) (codepoint 78, #o116, #x4e)
    preferred charset: ascii (ASCII (ISO646 IRV))
code point in charset: 0x4E
               script: latin
               syntax: w 	which means: word
             category: .:Base, L:Left-to-right (strong), U:Uppercase, a:ASCII, l:Latin, r:Roman
             to input: type "C-x 8 RET 4e" or "C-x 8 RET LATIN CAPITAL LETTER N"
          buffer code: #x4E
            file code: #x4E (encoded by coding system utf-8-unix)
              display: by this font (glyph code)
    xft:-ADBO-Source Code Pro-bold-normal-normal-*-18-*-*-*-m-0-iso10646-1 (#x0F)

Character code properties: customize what to show
  name: LATIN CAPITAL LETTER N
  general-category: Lu (Letter, Uppercase)
  decomposition: (78) ('N')

There are text properties here:
  face                 (:inherit hl-todo :foreground "#afd8af")
  fontified            t
  org-category         "todo"
```

*Expected behaviour:* :heart: :smile:
`DONE` text is highlighted the same in both cases.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 25.3.1
- Spacemacs: 0.200.10
- Spacemacs branch: master (rev. 4bb4cb46)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(helm emacs-lisp org)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GCONF GSETTINGS NOTIFY ACL GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 MODULES

**Comment by Glumanda99** [comment]
I also had this issue at least once the last days, just saying. Not being able to reproduce.

**Comment by Alexander-Miller** [comment]
Looks like it's due to hl-todo being active in org-mode when it really shoudl't be. [This hook](https://github.com/syl20bnr/spacemacs/blob/master/layers/%2Bspacemacs/spacemacs-ui-visual/packages.el#L159) needs to be changed to something that excludes org-mode.

**Comment by Alexander-Miller** [comment]
Nitpick: hooks should be named functions, so you can change or remove them again. That final nil isn't needed either.

**Comment by FryJay** [comment]
I've run into this issue as well when looking at org files.  It ignores some of the colors I've set in org-todo-keyword-faces but they show up correctly in agendas.  Adding the line @dudelson posted seems to have fixed it for me.

**Comment by duianto** [comment]
The following PR is now on the `develop` branch:
Fix #9950: Hook added to disable hl-todo-mode in org files #11732
